src/modules/users/services/users.service.spec.ts

```diff
@@ -34,11 +50,29 @@ describe('UsersService', () => {
 		it('should get a user by identity id', async () => {
 			//data.users_profile[0]
 			mockDataService.execute.mockReturnValueOnce({
-				data: { users_profile: [{ id: '123' }] },
+				data: { users_profile: [graphQlUserResponse] },
 			});
 
 			const result = await usersService.getUserByIdentityId('123');
-			expect(result).toEqual({ id: '123' });
+			expect(result).toEqual(archiefUser);
+		});
+
+		it('throws a notfoundexception if the user was not found', async () => {
+			mockDataService.execute.mockResolvedValueOnce({
+				data: {
+					users_profile: [],
+				},
+			});
+			let error;
+			try {
+				await usersService.getUserByIdentityId('unknown-id');
+			} catch (e) {
+				error = e;
+			}
+			expect(error.response).toEqual({
+				message: 'Not Found',
```

Comment:
A little bit more specific error message, otherwise it is hard to know if the route was not found or the user.
```suggestion
				message: 'User with id was not found',
```